### PR TITLE
CTW-733 Notification badge styling

### PR DESCRIPTION
**Pull request by eric-gunther**
Added new color palette in our theming for "notifications" (discussed with Tai). Defaulting to the same colors as our primary color palette, but for Canvas we will be setting to red/white per their theming needs.

image

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -24,6 +24,16 @@ const {
...
+// Feel free to play with this theme object
+const theme = {
+  colors: {
+    notification: {
+      main: "#FFFFFF",
+      light: "#BD0B00",
+    },
+  },
+};
+
```
Comment (eric-gunther) [inline src/App.tsx:36]: Would like to leave this here for testing/prototyping.

**Review by luthfib — DISMISSED** [review]

**Comment by eric-gunther** [comment]
@luthfib I forgot changeset :(

**Review by pmadria3 — APPROVED** [review]
LGTM

File: .changeset/healthy-gorillas-attack.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@zus-health/ctw-component-library": patch
+---
+
+Added notification color palette for badge
```
Comment (luthfib) [inline .changeset/healthy-gorillas-attack.md:5]:
```suggestion
Add notification color palette for badge.
```

**Review by luthfib — APPROVED** [review]

**Review by mrosata — APPROVED** [review]

**Pull request merged**